<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationWindow</class>
 <widget class="QDialog" name="RegistrationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>240</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Password 2:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MakeRegPushButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Make Reg</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PassworOneLineEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>160</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Login:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelPushButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>300</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LoginLineEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>200</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordTwoLineEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Visible password</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
